package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "enhanced-resolve",
   "version": "5.14.1",
+  "packageManager": "yarn@1.22.19",
```

Comment:
Let's avoid it here

Reply:
@merceyz I'm available to help get this through PR through now...

Do we need to specify `packageManager` in the `package.json` for `enhanced-resolve`? Does that help with the resolving logic?

Reply:
Don't strictly need to but since the version of Yarn used by this project isn't documented I guessed and documented it.